### DISCUSSION TITLE
求问 js 里面连续切换两个PAG 播放卡顿问题怎么解？

### DISCUSSION THREAD

**Discussion by yufengvac** [Tencent/libpag#2348] [Q&A]
https://github.com/Tencent/libpag/assets/7565523/96a07472-5411-4876-8a07-c527ac188b61

如视频所示，我使用 Vue 加载两个 PAG，来回切换大概七八次之后发现 PAG 播放极其卡顿，我怀疑是内存没有释放，请问下 vue 里面的 PAG 如何释放内存呀？（貌似在 vue 里面无法在外面声明 pagView 变量，然后在 beforeMount 调用 destroy 方法）

代码是：
<img width="920" alt="image" src="https://github.com/Tencent/libpag/assets/7565523/635b3e90-f049-44bd-bb63-4d7a52e7a196">
<img width="998" alt="image" src="https://github.com/Tencent/libpag/assets/7565523/286f6986-07f4-4c7f-b56a-df89ab740ce5">

源码：[vue3.zip](https://github.com/user-attachments/files/15951227/vue3.zip)

**Comment by CodeJhF** [comment]
同类问题已由 #2701、https://github.com/Tencent/libpag/discussions/3002 继续跟踪，为避免重复排查，先关闭本条。
如果有新的复现信息，欢迎到该帖补充，或在本帖回复后重开。（stale-close）
